test/Microsoft.ML.OnnxTransformTest/OnnxTransformTests.cs

```diff
@@ -0,0 +1,215 @@
+                        {
+                            sum += val;
+                            if (i == 0)
+                                Assert.InRange(val, 0.00004, 0.00005);
+                            if (i == 1)
+                                Assert.InRange(val,   0.003844, 0.003845);
+                            if (i == 999)
+                                Assert.InRange(val, 0.0029566, 0.0029567);
+                            i++;
+                        }
+                    }
+                    Assert.InRange (sum, 1.0, 1.00001);
+                }
+            }
+        }
+
+        [Fact]
+        public void OnnxStatic()
+        {
+            if (RuntimeInformation.IsOSPlatform(OSPlatform.Linux))
+                return;
+
+            var modelFile = "squeezenet/00000001/model.onnx";
+
+            using (var env = new ConsoleEnvironment(null, false, 0, 1, null, null))
+            {
+                var imageHeight = 224;
+                var imageWidth = 224;
+                var dataFile = GetDataPath("images/images.tsv");
+                var imageFolder = Path.GetDirectoryName(dataFile);
+
+                var data = TextLoader.CreateReader(env, ctx => (
+                    imagePath: ctx.LoadText(0),
+                    name: ctx.LoadText(1)))
+                    .Read(new MultiFileSource(dataFile));
+
+                // Note that CamelCase column names are there to match the TF graph node names.
+                var pipe = data.MakeNewEstimator()
+                    .Append(row => (
+                        row.name,
+                        data_0: row.imagePath.LoadAsImage(imageFolder).Resize(imageHeight, imageWidth).ExtractPixels(interleaveArgb: true)))
+                    .Append(row => (row.name, softmaxout_1: row.data_0.ApplyOnnxModel(modelFile)));
+
+                TestEstimatorCore(pipe.AsDynamic, data.AsDynamic);
+
+                var result = pipe.Fit(data).Transform(data).AsDynamic;
+                result.Schema.TryGetColumnIndex("softmaxout_1", out int output);
+                using (var cursor = result.GetRowCursor(col => col == output))
+                {
+                    var buffer = default(VBuffer<float>);
+                    var getter = cursor.GetGetter<VBuffer<float>>(output);
+                    var numRows = 0;
+                    while (cursor.MoveNext())
+                    {
+                        getter(ref buffer);
+                        Assert.Equal(1000, buffer.Length);
+                        numRows += 1;
+                    }
+                    Assert.Equal(3, numRows);
+                }
+            }
+        }
+
+        [Fact]
+        void TestCommandLine()
+        {
+            using (var env = new ConsoleEnvironment())
+            {
+                Assert.Equal(Maml.Main(new[] { @"showschema loader=Text{col=a:R4:0-3 col=b:R4:0-3} xf=OnnxTransform{inputs=a inputs=b outputs=c model={model_matmul/frozen_saved_model.pb}} in=f:\2.txt" }), (int)0);
```

Comment:
>inputs=a inputs=b outputs=c [](start = 116, length = 27)

in your argument class you have this:
public string InputColumn;
public string OutputColumn

you accept only one column as input and one as output.
also names are different.